### ISSUE TITLE
Getting 404 for empty input

### ISSUE THREAD

**Issue by Illusion47586** [pastelsky/bundlephobia#770]
**Describe the bug**
It looks like it is expected behaviour but I don't think we should allow an empty input call from the frontend.

**To Reproduce**
<img width="741" alt="image" src="https://user-images.githubusercontent.com/51574716/229738659-298ef259-efb6-41d3-8020-8048bc7523de.png">
Don't type anything in the search box and press enter.
<img width="1201" alt="image" src="https://user-images.githubusercontent.com/51574716/229738849-3cfd1444-a4bb-43bf-b59f-b905cdf4313b.png">
It takes you to `https://bundlephobia.com/package` which gives 404.

**Expected behavior**
I believe we should just do nothing when the input is empty.

Let me know if the suggested fix is good and I can make a relevant PR for the same.

**Comment by pastelsky** [comment]
Makes sense — the fix should be quite simple, so marking this as a good first issue.
